2018-06-26 23:00:23,517 [        MainThread][         start][    INFO] Starting TheRaidMap
2018-06-26 23:00:23,519 [        MainThread][       _legacy][    INFO] Starting factory <client.VNCDoToolFactory instance at 0x10a851cf8>
2018-06-26 23:00:23,701 [            VNCPic][       _legacy][    INFO] Using protocol version 3.800
2018-06-26 23:00:26,226 [        MainThread][   detect_text][    INFO] Check for Loginbutton - start ...
2018-06-26 23:00:26,530 [        MainThread][   detect_text][    INFO] Check for Loginbutton - finished...
2018-06-26 23:00:26,531 [        MainThread][   detect_text][    INFO] Check for Closebutton - start ...
2018-06-26 23:00:26,860 [        MainThread][   detect_text][    INFO] w
2018-06-26 23:00:26,861 [        MainThread][         start][   ERROR] Uncaught exception
Traceback (most recent call last):
  File "start.py", line 104, in <module>
    main()
  File "start.py", line 68, in main
    check_message()
  File "/Users/fameitelhuber/Desktop/TheRaidMapper/detect_text.py", line 60, in check_message
    if timer2.len > 5:
AttributeError: 'unicode' object has no attribute 'len'
